parser_test.go

```diff
@@ -307,12 +302,13 @@ func testParse(t *testing.T, spec *Spec) {
 }
 
 func TestFile(t *testing.T) {
-	if testFile == "" {
+	flag.Parse()
+	if len(*testFile) == 0 {
```

Comment:
Maybe flag behavior on test changed at [this commit](https://github.com/golang/go/commit/fbc6a972226f889d2ab1150468755615098ee80f).

So call flag.Parse on out init phase, fail to parse flags declared on test package (such as `-test.testlogfile`).
I think need to change only flag.Prase position. Would you check it ?

Reply:
Thank you!
Certainly, it's need to change only flag.Prase position.

So, I changed at 92da2f6c8bb1d526d06a745e0f93266faa85e7a8